packages/react/src/components/filepicker/tests/FilepickerSelectors.spec.ts

```diff
@@ -3,7 +3,7 @@ import {FilepickerSelectors} from '../FilepickerSelectors';
 
 describe('FilepickerSelectors', () => {
     describe('getFile', () => {
-        // This selector cannot be tested, because for security reasons the browser won't allow to select a file programmatically.
+        // tested via the FilePicker component tests
```

Comment:
The 2 messages give relevant information, couldn't we merge the 2 into one?

Reply:
Well the first statement is made false by the second one 🤔  It CAN be tested, because it is tested in the component tests

Reply:
Ignoring your change, can this selector now be tested here? or do we still have the same problem?

If we still have the same problem, then having the information from the first comment is relevant, otherwise we don't care and we keep your change. ;)

Reply:
I think it's not relevant to test selectors on itself, they don't mean anything to a user.
I'm fine that it's tested ''pas la bande'' by the component 👍